fdbcli/tests/metacluster_fdbcli_tests.py

```diff
@@ -121,61 +113,70 @@ def configure_tenant(management_cluster_file, data_cluster_files, tenant, tenant
     if assigned_cluster:
         command = command + ' assigned_cluster={}'.format(assigned_cluster)
 
-    output, err = run_fdbcli_command_and_get_error(management_cluster_file, command)
+    _, output, err = run_fdbcli_command(management_cluster_file, command)
     return output, err
 
 
-@enable_logging()
-def clear_database_and_tenants(logger, management_cluster_file, data_cluster_files):
+def clear_database_and_tenants(management_cluster_file, data_cluster_files):
     subcmd1 = 'writemode on'
     subcmd2 = 'option on SPECIAL_KEY_SPACE_ENABLE_WRITES'
     subcmd3 = 'clearrange ' '"" \\xff'
     subcmd4 = 'clearrange \\xff\\xff/management/tenant/map/ \\xff\\xff/management/tenant/map0'
-    output = run_fdbcli_command(management_cluster_file, subcmd1, subcmd2, subcmd3, subcmd4)
-    logger.debug(output)
-
-
-def run_tenant_test(management_cluster_file, data_cluster_files, test_func):
-    test_func(management_cluster_file, data_cluster_files)
-    clear_database_and_tenants(management_cluster_file, data_cluster_files)
+    run_fdbcli_command(management_cluster_file, subcmd1, subcmd2, subcmd3, subcmd4)
 
 
 @enable_logging()
-def clusters_status_test(logger, cluster_files):
+def clusters_status_test(logger, cluster_files, max_tenant_groups_per_cluster):
+    logger.debug('Start')
```

Comment:
This should be removed or be more descriptive, since we are already logging "Tests start" below